overlord/snapstate/handlers.go

```diff
@@ -1006,6 +1056,32 @@ func writeSeqFile(name string, snapst *SnapState) error {
 	return osutil.AtomicWriteFile(p, b, 0644, 0)
 }
 
+// ComputeMissingDisabledServices returns a list of services that were disabled
+// that are currently missing from the specific snap info (i.e. they were
+// renamed in this snap info), as well as a list of disabled services that are
+// present in this snap info
+// note that the union of the missing and found svc lists should be exactly
+// equal to st.LastActiveDisabledServices
+func ComputeMissingDisabledServices(st *SnapState, info *snap.Info) ([]string, []string, error) {
```

Comment:
in context the code would be a bit more readable I think if this took st.LastDisabledServices explicity instead of snaps

Reply:
Okay, this change would also make this more easily unit testable

Reply:
I changed this in the other PR, I think I will wait to merge in those changes until that PR is merged to master to prevent force pushing or lots of extraneous merge commits.